tidb-cloud/migration-job.md

```diff
@@ -0,0 +1,161 @@
+
+To solve this problem, TiDB Cloud provides the migration job feature, which supports full migration and incremental migration, allowing you to migrate your business from the upstream database to TiDB Cloud within a short downtime window.
+
+A migration job supports the following data sources:
+
+- MySQL 5.6-8.0 on premises or on a cloud. Note that MySQL 8.0 is still experimental and might have incompatible issues.
+- AWS Aurora MySQL 5.6 and 5.7
+- AWS RDS MySQL 5.7
+
+A migration job supports the following network types:
+
+- Public IP
+- VPC Peering
+- Private Link
+
+A migration job supports data migration within the same region and cross regions.
+
+## Limitations
+
+- The system databases including `mysql`, `information_schema`, `information_schema` and `sys` will be filtered out during migration (even when you migrate the entire upstream MySQL instance), and will not be migrated to TiDB Cloud.
+
+- Currently the migration job feature is only deployed on the us-west-2 region and the EKS provider, which means that the **Migration Job** tab will not be displayed in old clusters created on AWS.
+
+- If the table to migrated already exists in the target database, TiDB Cloud appends the data to the target table directly. If the keys conflict, it will report an error.
+
+- When you delete a cluster, all migration jobs in that cluster are automatically deleted, and the deleted migration jobs are not recoverable.
+
+- Currently the migration job feature is still in public beta. During public beta, each account can create one migration job.
+
+## Prerequisites
+
+### Privileges for the upstream database
+
+The username you use for the upstream database must have the following privileges:
+
+| Privilege | Scope |
+|:----|:----|
+| `SELECT` | Tables |
+| `RELOAD` | Global |
+| `REPLICATION SLAVE` | Global |
+| `REPLICATION CLIENT` | Global |
+| `PROCESS` | Global |
+
+For example, you can execute the following `GRANT` statement grant privileges:
+
+```sql
+GRANT SELECT,RELOAD,REPLICATION SLAVE,REPLICATION CLIENT,LOCK TABLES,PROCESS ON *.* TO 'your_user'@'your_wildcard_of_host'
+```
+
+If you also need to migrate the data from other databases to TiDB Cloud, make sure the same privileges are granted to the user of the respective databases.
+
+### Privileges for the downstream TiDB Cloud cluster
+
+The username you use for the downstream TiDB Cloud cluster must have the following privileges:
+
+| Privilege | Scope |
+|:----|:----|
+| `CREATE` | Tables |
+| `SELECT` | Tables |
+| `INSERT` | Tables |
+| `UPDATE` | Tables |
+| `DELETE` | Tables |
+| `ALTER`  | Tables |
+| `DROP`   | Tables |
+| `INDEX`  | Tables |
+
+For example, you can execute the following `GRANT` statement grant privileges:
+
+```sql
+GRANT CREATE,SELECT,INSERT,UPDATE,DELETE,ALTER,DROP,INDEX ON *.* TO 'your_user'@'your_wildcard_of_host'
```

Comment:
Does "your_wildcard_of_host" mean the IP address of your host?

Reply:
yes updated.